src/views/Home/index.view.tsx

```diff
@@ -1,15 +1,10 @@
 import * as React from "react"
 import "./index.scss"
-import { useAuth0 } from "@auth0/auth0-react"
```

Comment:
Let's just add all the imports for any components that need to be contained in HomeView and place them the HomeView function